### ISSUE TITLE
Probably bug: Importing boardarchive from 0.16.0 to 7.0.0

### ISSUE THREAD

**Issue by alexmechanic** [mattermost/focalboard#3208]
## Steps to reproduce the behavior

1. Setup and run personal focalboard server version [0.16.0](https://github.com/mattermost/focalboard/releases/tag/v0.16.0)
2. Create any of: boards, templates, categories, etc.
3. Select Settings - Archive Export - save `*. boardarchive` file to disk
4. Setup and run personal focalboard server version [7.0.0](https://github.com/mattermost/focalboard/releases/tag/v7.0.0) with empty database (I was running postgres). 
5. Select Settings - Import - Archive - select saved `*. boardarchive` file

## Expected behavior

Terminal prompts error messages:
```
debug [2022-06-09 10:32:26.661 +03:00] importing archive                        caller="app/import.go:53"
error [2022-06-09 10:32:26.664 +03:00] API ERROR                                caller="api/api.go:1934" code=500 error="cannot import board b3xtj3eggkpnojqk7ie9hrj3bko: unsupported archive line type; got board, line 1" msg= api=/api/v1/workspaces/0/archive/import
```

## Screenshots

Nothing happens on web views.

## Edition and Platform

 - Edition: Personal Server
 - Versions: 7.0.0
 - Browser and OS: Ubuntu 20.04 LTS, tested on Chrome, Safari, Firefox.

## Additional context

Not sure if archive files should definitely be compatible between such versions, but discussing this with community showed that this might be important.

**Comment by wuwinson** [comment]
Thanks for reporting @alexmechanic! Unfortunately, `v0.16 RC1` was an interim release that we ended up not shipping, which is why you're running into this issue. Apologies for any confusion that may have caused. We removed that release to prevent others from running into the same issue. I've added [this discussion](https://github.com/mattermost/focalboard/discussions/3225) for additional clarity.

**Comment by alexmechanic** [comment]
Well, that is really sad as this version added new boards permissions, which I definitely needed.

**Comment by wuwinson** [comment]
@alexmechanic Commented on https://github.com/mattermost/focalboard/discussions/3225, but just wanted to clarify here that the new permissions system will be available in `v7.2`!
